src/libs/SidebarUtils.ts

```diff
@@ -117,7 +117,7 @@ function getOrderedReportIDs(
     betas: Beta[],
     policies: Record<string, Policy>,
     priorityMode: ValueOf<typeof CONST.PRIORITY_MODE>,
-    allReportActions: OnyxCollection<ReportActions>,
+    allReportActions: OnyxCollection<ReportAction[]>,
```

Comment:
I'm curious about this change, the structure should be `OnyxCollection<ReportActions>`.

Reply:
Yes, but in the component where this method is used , there is a selector in `withOnyx` which is mapping value to type `ReportAction[]`

Reply:
In the original issue I put a screenshot (I am also a creator of this issue XD) I found it when migrating ReportUtils